app/src/main/java/kumamotone/sample/eremotethingssample/MainActivity.kt

```diff
@@ -71,40 +72,20 @@ class MainActivity : Activity() {
         }
     }
 
-    private fun oyasumi() {
-        launch {
-            async {
-                sendToRemoconWithoutCoroutine(getString(R.string.light_off))
-            }.await()
-            async {
-                sendToRemoconWithoutCoroutine(getString(R.string.aircon_off))
-            }.await()
-            async {
-                sendToRemoconWithoutCoroutine(getString(R.string.timer7hours))
-            }
-        }
+    private suspend fun oyasumi() {
+        sendToRemocon(getString(R.string.light_off))
+        sendToRemocon(getString(R.string.aircon_off))
+        sendToRemocon(getString(R.string.timer7hours))
```

Comment:
We can call `sendRemocon` suspending function just in order.
Because suspending function is invoked sequentially.

Reply:
Okay 👍